src/hooks/options-validation.ts

```diff
@@ -1,9 +1,9 @@
-const { default: ow } = require('ow');
+import ow from 'ow';
 
 /**
  * @param {object} options
  */
-function optionsValidationHandler(options) {
+export function optionsValidationHandler(options: unknown): void {
```

Comment:
The possible options are quite well known, no?

Reply:
They aren't known upon validation. We could type `asserts options it T` but it still doesn't matter as this is a hook.